src/openfermion/utils/_trotter_qasm_conversion.py

```diff
@@ -0,0 +1,331 @@
+#	Licensed under the Apache License, Version 2.0 (the "License");
+#	you may not use this file except in compliance with the License.
+#	You may obtain a copy of the License at
+#
+#		http://www.apache.org/licenses/LICENSE-2.0
+#
+#	Unless required by applicable law or agreed to in writing, software
+#	distributed under the License is distributed on an "AS IS" BASIS,
+#	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#	See the License for the specific language governing permissions and
+#	limitations under the License.
+
+from openfermion.ops import QubitOperator
+import numpy as np
+
+'''
+Description:
+		Approximates the exponential of an operator composed of Pauli
+		matrices, by Trotterization. Outputs QASM format to a python
+		stream.
+
+		Change-of-basis gates:
+		H			: Z to X basis (and back)
+		Rx(pi/2)	: Z to Y basis ..... 
+		Rx(-pi/2) 	: Y to Z basis ..... 
+
+'''
+
+
+
+
+def third_order_trotter_helper(hamiltonian_dict,trotterized_list,op_a, op_b):
+	'''
```

Comment:
The docstrings should follow a particular standard as it helps both with automatic doc generation for the website and consistency thorughout the code.  It's easiest to look at some of the other functions to get a feeling for what it is.